### PR TITLE
Global help button

### PR DESCRIPTION
**Pull request by Piotrek605**

### CODE REVIEW

File: meteor-app/imports/ui/layouts/full-window/style.less [hunk 1]
```diff
@@ -10,6 +10,8 @@
   align-items: stretch;
 
   .page__header {
+    .flex-row;
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/full-window/style.less]: The chunk here (line 5 to line 22 previously) described explicitly the flex relationship between the parent and the children. While the line you added here is only relevant to the child itself. It does not belong here.

File: meteor-app/imports/ui/layouts/full-window/style.less [hunk 2]
```diff
@@ -18,6 +20,8 @@
   }
 
   .page__footer {
+    .flex-row;
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/full-window/style.less]: Same with above.

**Comment by Zodiase** [comment]
Other than that LGTM.

File: meteor-app/imports/ui/layouts/full-window/style.less (hunk 1 above)
Comment (Piotrek605) [inline meteor-app/imports/ui/layouts/full-window/style.less]: Fixed.

File: meteor-app/imports/ui/layouts/full-window/style.less (hunk 2 above)
Comment (Piotrek605) [inline meteor-app/imports/ui/layouts/full-window/style.less]: Fixed.

**Pull request merged**